packages/@ionic/cli/src/lib/project/index.ts

```diff
@@ -563,11 +564,66 @@ export abstract class Project implements IProject {
 
     await writeJson(this.packageJsonPath, pkg, { spaces: 2 });
 
+    if (themeColor) {
+      await this.setTheme(themeColor);
+    }
+
     const integrations = await this.getIntegrations();
 
     await Promise.all(integrations.map(async i => i.personalize(details)));
   }
 
+  async setTheme(themeColor: string): Promise<void> {
+    let themePath;
+    switch (this.type) {
+      case 'angular': {
+        themePath = path.join(this.directory, 'src', 'theme', 'variables.scss');
+        break;
+      }
+      case 'react': {
+        themePath = path.join(this.directory, 'src', 'theme', 'variables.css');
+        break;
+      }
+    }
+    if (!themePath) {
+      return;
+    }
+
+    return this.writeThemeColor(themePath, themeColor);
+  }
```

Comment:
This likely belongs in the `project/angular` Project class, as it should only run for Angular projects.